templates/elements/journal/issue_top.html

```diff
@@ -3,7 +3,7 @@
 
 {% if issue.large_image %}
     <div class="card">
-        <img class="card-img img-fluid article-img" src="{{ issue.large_image.url }}" alt="">
+        <img class="card-img img-fluid article-img" src="{{ issue.large_image.url }}" alt="{{ issue.issue_title }} Cover Image">
```

Comment:
Should we use `issue.display_title` instead? For issues that don't have an explicit titles